ghost/collections/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@tryghost/collections",
+    "version": "0.0.0",
+    "repository": "https://github.com/TryGhost/Ghost/tree/main/packages/collections",
+    "author": "Ghost Foundation",
+    "private": true,
+    "main": "build/index.js",
+    "types": "build/index.d.ts",
+    "scripts": {
+        "dev": "tsc --watch --preserveWatchOutput --sourceMap",
+        "build": "tsc",
+        "prepare": "tsc",
+        "test:unit": "NODE_ENV=testing c8 --src src --all --exclude 'src/Collection.ts' --exclude 'test/' --check-coverage --100 --reporter text --reporter cobertura mocha -r ts-node/register './test/**/*.test.ts'",
```

Comment:
@allouis I didn't find a quick a better quick way to not count the file that only includes a type definition in the test coverage, so used this `--exclude 'src/Collection.ts'` explicit way as a stopgap. have you encountered a better way in the past?

Reply:
I ran into the same thing in simple-mfa; my solution was to have a dedicated folder for interfaces and [exclude](https://togithub.com/vikaspotluri123/simple-mfa/blob/e51a557bbe5cd95b1c5c685f8c93b21585c26025/.c8rc.json) it in my c8 config.

However, since your tests are written in typescript a possible solution is to use the types in the tests (e.g. explicitly typing a variable) - but that also depends on well ts-node and c8 work together 🙂

Reply:
Oh that's a good shout with a separate interfaces folder 🙌